Explorer/Assets/DCL/Infrastructure/ECS/SceneLifeCycle/IScenesCache.cs

```diff
@@ -7,10 +8,10 @@ namespace ECS.SceneLifeCycle
 {
     public interface IScenesCache
     {
+        event Action<ISceneFacade?>? OnCurrentSceneChanged;
```

Comment:
Lets rethink the idea of having a reactive action. We can have a call about it if you want, but I offer you some ideas:

1)  We intentionally added `ISceneIsCurrentListener`; maybe we can have a system that listens to the change?
2) Even better, since the time is a realm operation, I would suggest trying to add an operation in `RealmNavigator`; particularly in `realmChangeOperations`